eth-contracts/contracts/DelegateManager.sol

```diff
@@ -430,6 +404,49 @@ contract DelegateManager is RegistryContract {
         );
     }
 
+    function _validateClaimRewards(address _sp)
+    internal returns (uint totalBalanceInStaking, uint totalBalanceInSPFactory, uint totalBalanceOutsideStaking)
+        {
+
+        ServiceProviderFactory spFactory = ServiceProviderFactory(
```

Comment:
can we accept spFactory as an argument here...? that would avoid a dupe instantiation in a single function call

Reply:
good call @hareeshnagaraj. contracts compiled locally for this change. lets see if aall the tests pass https://github.com/AudiusProject/audius-protocol/pull/488/commits/c7c90137811164a3d96d36462324844c7df3c380